packages/connect-web/tsconfig.json

```diff
@@ -10,13 +10,6 @@
         ],
         "rootDir": ".."
     },
-    "include": [
-        "./connect-web",
-        "../connect-iframe/webpack/prod.webpack.config.ts",
-        "../connect-iframe/package.json",
-        "../connect-popup/webpack/prod.webpack.config.ts",
-        "../connect-popup/package.json"
-    ],
```

Comment:
this include statement was actually causing 
`yarn workspace @trezor/connect-web type-check` to do nothing. I am not sure how to fix this properly. Is there some typescript mage around?

Reply:
"./connect-web" > "." should do the trick

Reply:
forced push, lets see